common/supplemental/supplementalData.xml

```diff
@@ -1893,8 +1893,7 @@ XXX Code for transations where no currency is involved
 		<language type="peo" scripts="Xpeo"/>
 		<language type="pfl" scripts="Latn"/>
 		<language type="phn" scripts="Phnx"/>
-		<language type="pi" scripts="Mymr"/>
-		<language type="pi" scripts="Sinh Deva Thai" alt="secondary"/>
+		<language type="pi" scripts="Deva Mymr Sinh Thai" alt="secondary"/>
```

Comment:
do only the secondary scripts get listed in this section of the supplementalData?